build/Helix/GenerateTestProjFile.ps1

```diff
@@ -0,0 +1,282 @@
+Param(
+    [Parameter(Mandatory = $true)] 
+    [string]$TestFile,
+
+    [Parameter(Mandatory = $true)] 
+    [string]$OutputProjFile,
+
+    [Parameter(Mandatory = $true)] 
+    [string]$JobTestSuiteName,
+
+    [Parameter(Mandatory = $true)] 
+    [string]$TaefPath
+)
+
+Add-Type -Language CSharp -ReferencedAssemblies System.Xml,System.Xaml @"
+using System;
```

Comment:
IMHO I feel like this would be simpler if it was just powershell script. Since it's already written I won't ask you to rewrite it, but doing this kind of parsing and munging of some output is what powershell is pretty good at.

Reply:
Huh, I actually didn't know that a lot of this functionality was supported in PowerShell these days, like classes and enums.  It shouldn't be too hard to switch it over to PowerShell - that'll certainly make any debugging a lot easier.